src/types/@mui/material/styles.d.ts

```diff
@@ -0,0 +1,9 @@
+import type {CSSProperties} from '@mui/styles';
```

Comment:
We need to declare this module to tell MUI what custom properties we have in our Theme. This might be something we try to expect directly from SSC when we convert it to TS?